### DISCUSSION TITLE
Why is GrandOrgue expecting InfoFileName in html format ?

### DISCUSSION THREAD

**Discussion by eturpault** [GrandOrgue/OdfEdit#10] [Q&A] [answered]
With the Hauptwerk sample sets of Piotr Grabowski, the organ info file is a PDF file, this makes GrandOrgue to raise the following warning with ODF generated by OdfEdit : 
       Warning: InfoFileName does not point to a html file
Why is it an issue for GrandOrgue that the info file is not a html file ?

**Comment by larspalo** [comment, marked as answer]
InfoFileName is another property that's basically only supported for legacy reasons and compatibility with HW1 style sample sets. It's currently not used in any way from inside GO so it's in a way better to leave it out of the ODF completely.

**Reply by eturpault** [reply]
Ok thanks for the clarification, so I will let it empty for the next OdfEdit releases.
